features/schedule/trigger/feature.go

```diff
@@ -0,0 +1,58 @@
+package trigger
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/uuid"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows: Workflow,
+	Execute:   Execute,
+}
+
+func Workflow(ctx workflow.Context, arg string) (string, error) { return arg, nil }
+
+func Execute(ctx context.Context, r *harness.Runner) (client.WorkflowRun, error) {
+	// Create paused 1m schedule
+	workflowID := uuid.NewString()
+	handle, err := r.Client.ScheduleClient().Create(ctx, client.ScheduleOptions{
+		ID:   uuid.NewString(),
+		Spec: client.ScheduleSpec{Intervals: []client.ScheduleIntervalSpec{{Every: 1 * time.Minute}}},
+		// TODO(cretz): Change to pointer, see https://github.com/temporalio/sdk-go/issues/958
+		Action: client.ScheduleWorkflowAction{
+			ID:        workflowID,
+			Workflow:  Workflow,
+			Args:      []interface{}{"arg1"},
+			TaskQueue: r.TaskQueue,
+		},
+		Paused: true,
+	})
+	r.Require.NoError(err)
+
+	// Remove the schedule on complete
+	defer func() {
+		if err := handle.Delete(context.Background()); err != nil {
+			r.Log.Warn("Failed deleting schedule handle", "error", err)
+		}
+	}()
+
+	// Trigger it
+	r.Require.NoError(handle.Trigger(ctx, client.ScheduleTriggerOptions{}))
+	// TODO(cretz): We have to wait before triggering again. See
+	// https://github.com/temporalio/temporal/issues/3614
+	time.Sleep(2 * time.Second)
+	r.Require.NoError(handle.Trigger(ctx, client.ScheduleTriggerOptions{}))
```

Comment:
Are we going to test different overlap options or do we consider that out of scope for this testing?

Reply:
Out of scope for just trigger testing. But if we need to test how overlap behaves in a test we can. However there are dozens of fields in schedule we are not testing here in SDK features. I am unsure who is the arbiter of what we should or shouldn't test.

For example, how do we know buffer one works properly with the Go SDK? I can't find an integration test with it.